HopsanGUI/Utilities/GUIUtilities.cpp

```diff
@@ -377,18 +377,24 @@ QVector< complex<double> > realToComplex(const QVector<double> &rRealVector)
 //! @brief Apply windowing function
 //! Only Hann windows are supported for now
 //! @param data Vector with data to be windowed
-void windowFunction(QVector<double> &data, WindowingFunctionEnumT function)
+//! @param Ca Gain compensation factor
+//! @param Cb Frequency compensation factor
+void windowFunction(QVector<double> &data, WindowingFunctionEnumT function, double &Ca, double &Cb)
```

Comment:
Not super important but if Ca and Cb are "return value parameters" they should be called rCa and rCb
Also doxygen @param should add [out]
@param [out] Ca